### DISCUSSION TITLE
Make conversions involving multiple units.

### DISCUSSION THREAD

**Discussion by awa5114** [angularsen/UnitsNet#1228] [Q&A] [answered]
Is it possible to convert from, say m3/s to L/day using this library?

**Comment by angularsen** [comment, marked as answer]
Yes, if there are defined units with those abbreviations, then you can.

```cs
VolumeFlow.Parse("5 m³/s").ToUnit(VolumeFlow.ParseUnit("L/day"))
```

Note that `m3/s` is not defined, but `m³/s` is. If needed, you can add the `m3/s` unit abbreviation too.
https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit
